mu/locale/es/LC_MESSAGES/mu.po

```diff
@@ -5,756 +5,1091 @@
 msgid ""
```

Comment:
Not sure what to do with the header copyright on the second line, should I add anything specific to the Mu project?

Reply:
Sounds good. Looking at other files this is the current copyright line:

```
Copyright (c) 2015-2017 Nicholas H.Tollervey and others (see the AUTHORS file).
```

Should I use the same with the updated year?

Reply:
Sounds like a plan! :-)